addons/actions.md

```diff
@@ -168,6 +168,111 @@ end
 
 For information on making use of the [openHAB Cloud service](https://github.com/openhab/openhab-cloud/blob/master/README.md) hosted by the [openHAB Foundation e.V.](https://www.openhabfoundation.org/), visit the [myopenhab.org website](http://www.myopenhab.org).
 
+## Ephemeris
+
+Ephemeris is a way to determine what type of day today or a number of days before or after today is. For example, to determine if today is a weekend, a bank holiday, someone’s birthday, trash day, etc.
+The default bank holidays and configuration syntax is defined by the [Jollyday library](https://github.com/svendiedrichsen/jollyday).
+
+### Actions
+
+#### Rules DSL
+
+- `getBankHolidayName`: returns the name of the holiday today, or `null` if today is not a bank holiday.
```

Comment:
When i take a look at the preview i would say this is a perfect usecase for a markdown table.
Every explanation begins with `returns`. This should be a coloumn header.

Reply:
Will be done in the upcoming commit.